src/global/Descriptor.java

```diff
@@ -0,0 +1,47 @@
+package global;
+
+public class Descriptor {
+    
+    int[] value;
+    	
+    	Descriptor() {
+        	value = new int[5];
+    		value[0] = 0;
+    	    value[1] = 0;
+    	    value[2] = 0;
+    	    value[3] = 0;
+    	    value[4] = 0;
+    	}
+    
+    void set(int value0,int value1, int value2, int value3, int value4) {
+        value[0] = value0;
+        value[1] = value1;
+        value[2] = value2;
```

Comment:
requirement says, value will be between 0 to 10000, do a check and throw an exception if anybody tries to set a values outside this range.